File: Python/3289_the_two_sneaky_numbers_of_digitville.py
class Solution:
    def getSneakyNumbers(self, nums: list[int]) -> list[int]:
        seen = set()
        ret_list = []
        for num in nums:
            if num not in seen:
                seen.add(num)
            else:
                ret_list.append(num)
        return ret_list

File: JavaScript/3289_the_two_sneaky_numbers_of_digitville.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var getSneakyNumbers = function(nums) {
    let ret = [];
    nums.sort();
    for (let i = 1; i < nums.length; i++) {
        if (nums[i] === nums[i - 1]) {
            ret.push(nums[i]);
        }
    }
    return ret;
};

